### PR TITLE
MODEUR-14: Wait until POST /_/tenant completes

### PR DESCRIPTION
**Pull request by julianladisch**
This fixes failures like this:
https://jenkins-aws.indexdata.com/blue/organizations/jenkins/folio-org%2Fmod-eusage-reports/detail/master/46/pipeline
https://jenkins-aws.indexdata.com/job/folio-org/job/mod-eusage-reports/job/MODEUR-14-book-report/2/execution/node/53/log/

### CODE REVIEW

File: src/test/java/org/folio/tlib/api/ApiIT.java [hunk 1]
```diff
@@ -94,12 +95,30 @@ public void health() {
...
+        throw new AssertionFailedError("POST /_/tenant timeout");
+      }
+      complete =
+          when().
+            get(location + "?wait=1").
+          then().
+            statusCode(200).
+          extract().
+            path("complete");
+    } while (! complete);
```
Comment (adamdickmeiss) [inline src/test/java/org/folio/tlib/api/ApiIT.java:122]: wait=30 waits will only wait as long as it needs to ..

https://github.com/folio-org/mod-eusage-reports/blob/84ee318bb6ac794009e87cb54d8d3111d7513596/src/main/java/org/folio/tlib/api/Tenant2Api.java#L73

so you don't need a loop. ``?wait=30`` should suffice

**Review by adamdickmeiss — CHANGES_REQUESTED** [review]

**Review by adamdickmeiss — COMMENTED** [review]
Comment

**Review by adamdickmeiss — APPROVED** [review]
Comment (not a show-stopper to use a loop for this, but not needed)

File: src/test/java/org/folio/tlib/api/ApiIT.java (hunk 1 above)
Comment (julianladisch) [reply src/test/java/org/folio/tlib/api/ApiIT.java:112]: Yes, after fixing the milliseconds vs seconds bug: https://github.com/folio-org/mod-eusage-reports/pull/49/commits/5c643076870fa41cc0b567c4135c35bbee033e34

**Pull request merged**